products/batch_exports/backend/temporal/spmc.py

```diff
@@ -1259,6 +1259,13 @@ async def track_iteration_of_record_batches():
 
                 self.rows_exported_counter.add(num_records_in_batch)
 
+                self.logger.info(
+                    f"Consuming {num_records_in_batch:,} records, {num_bytes_in_batch / 1024**2:.2f} MiB from record "
+                    f"batch {total_record_batches_count}. Total records so far: {total_records_count:,}, "
+                    f"total MiB so far: {total_record_batch_bytes_count / 1024**2:.2f}, "
+                    f"total file MiB so far: {total_file_bytes_count / 1024**2:.2f}"
+                )
```

Comment:
Care to use %-formatting to avoid the computation if we don't log at info level?